documentation/modules/configuring/con-config-mirrormaker2-acls.adoc

```diff
@@ -9,7 +9,7 @@
 When using MirrorMaker 2 with Strimzi, it is possible to synchronize ACL rules for remote topics. 
 However, this feature is only available if you are not using the User Operator.
 
-If you are using the AclAuthorizer without the User Operator, the ACL rules that manage access to brokers also apply to remote topics. 
+If you are using the `AclAuthorizer` without the User Operator, the ACL rules that manage access to brokers also apply to remote topics. 
```

Comment:
MAybe better like this since the class name differs in ZooKeeper and in KRaft mode.

```suggestion
If you are using the `type: simple` authorization without the User Operator, the ACL rules that manage access to brokers also apply to remote topics. 
```